documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -189,3 +182,48 @@ combination with ``std::vector::size()`` as shown in the example above, rather t
 be possible for |onedpl_short| to detect that iterators are pointing to USM-allocated data. In that
 case the data will be treated as if it were host-allocated, with an extra copy made to a SYCL buffer.
 Retrieving USM pointers from ``std::vector`` as shown guarantees no unintended copying.
+
+Use Range Views
+---------------
+
+For :doc:`parallel range algorithms <parallel_range_algorithms>' with device execution policies,
+place the data in USM or a USM-allocated ``std::vector``, and pass it to an algorithm
+via a device-copyable range or view object such as ``std::ranges::subrange`` or ``std::span``.
+
+.. note::
+   Use of ``std::ranges::views::all`` is not supported for algorithms with device execution policies.
+
+These data ranges as well as supported range adaptors and factories may be combined into
+data transformation pipelines that also can be used with parallel range algorithms. For example:
+
+.. code:: cpp
+
+  #include <oneapi/dpl/execution>
+  #include <oneapi/dpl/algorithm>
+  #include <random>
+  #include <vector>
+  #include <sycl/sycl.hpp>
+
+  int main(){
+    const int n = 1000;
+    auto policy = oneapi::dpl::execution::dpcpp_default;
+    sycl::queue q = policy.queue();
+
+    int* d_head = sycl::malloc_host<int>(n, q);
+    std::generate(d_head, d_head + n, std::minstd_rand{});
+
+    sycl::usm_allocator<int, sycl::usm::alloc::shared> alloc(q);
+    std::vector<int, decltype(alloc)> vec(n, alloc);
+
+    oneapi::dpl::ranges::copy(policy,
+        std::ranges::subrange(d_head, d_head + n) | std::views::transform(std::negate),
+        std::span(vec));
+
+    oneapi::dpl::ranges::sort(policy, std::span(vec));
+
+    sycl::free(d_head, q);
+    return 0;
+  }
+
+:: _`SYCL buffer`: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#subsec:buffers
+:: _`unified shared memory`: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:usm
```

Comment:
Something is wrong with the rendering in the generated documentation:

![image](https://github.com/user-attachments/assets/3e8de085-37f6-4941-8248-b4c2c8980c39)


The lines should start with `..` instead of `::` if these suppose to be footnotes.

Reply:
Thanks, fixed.